### PR TITLE
Added star rating feature

### PR DESCRIPTION
**Pull request by PKartavkin**

### CODE REVIEW

**Comment by PKartavkin** [comment]
@HristiyanZahariev Can you take a look on it and help me with labels placing? The logic is done, but I have issues with placing labels (xcode doesn't show them, but when app is running everything is ok).

**Comment by HristiyanZahariev** [comment]
Do you mean the numbers on the right? You want them to be aligned to the right or what? I can also refactor this code.

**Comment by HristiyanZahariev** [comment]
Ok I fixed the counter labels but I don't know why the xcode doesn't update when doing ui stuff on the add idea screen. Maybe I should investigate this, its not normal behavior

**Comment by PKartavkin** [comment]
I've sent invite for this repo

File: BrainStormApp/Controllers/AddIdeaViewController.swift [hunk 1]
```diff
@@ -40,17 +41,33 @@ class AddIdeaViewController: UIViewController {
...
+        
+        timeToMarketCosmosView.rating = 5
+        requiredMoneyCosmosView.rating = 5
+        profitCosmosView.rating = 5
+        difficultyCosmosView.rating = 5
+        
+        timeToMarketCosmosView?.didTouchCosmos =
+            {(rating:Double)->() in
+                self.timeToMarketCosmosView.rating = rating.rounded(.up)
+                self.timeToMarketLabel.text = String(Int(rating.rounded(.up)))}
```
Comment (PKartavkin) [inline BrainStormApp/Controllers/AddIdeaViewController.swift]: Perhaps it is possible to make closure for this and reuse it

**Pull request merged**